auacm/app/static/js/controllers/ScoreboardController.js

```diff
@@ -38,26 +37,26 @@ app.controller('ScoreboardController', ['$scope', '$http', '$routeParams',
         var prevSolved = $scope.teams[0].solved;
         var prevTime = $scope.teams[0].solved;
         $scope.teams[0].rank = rank;
-        for (var i = 1; i < $scope.teams.length; i++) {
-            var team = $scope.teams[i];
-            if (team.solved < prevSolved) {
+        for (var j = 1; j < $scope.teams.length; j++) {
```

Comment:
[Apparently](https://stackoverflow.com/questions/26174520/reusing-i-for-javascript-for-loops) JavaScript is *function-level scoped* (I think) so the `i` stays in scope after the first `for` loop. Therefore, the second loop is actually _redeclaring_ `i`. However, redeclaring doesn't throw a runtime error, just a linting error. IMO, I'd rather keep it consistent and deal with/suppress the linter errors; my reason is that when I see `j` as a counter variable, it looks like it's a nested loop, which it's not.

***tl;dr*** JavaScript is weird.